build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
     id("java")
     id("checkstyle")
     id("maven-publish")
+    kotlin("jvm") version "2.0.0"
```

Comment:
remove this? if we wanted kotlin support, we'd remove the other plugin, right?

Reply:
Yes, I forgot that one after trying Kotlin.